### PR TITLE
created modal & slider components

### PR DESCRIPTION
**Pull request by agrawalshanu708**
preview : https://nervous-mcclintock-a6b287.netlify.app/

### CODE REVIEW

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      #modal-content{
+        border: 1px solid var(--primary-color);
+        width: 40rem;
+        padding: 1rem;
+        visibility: hidden;
+      }
+      .modal:active{
+        background-color: var(--primary-color);
+        color: var(--background-color);
+      }
```
Comment (bharati-21) [inline modal/modal.css:76]: The only suggestion is to look at code formatting. Could use consistent spacings and tabs 👍🏽

File: modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline modal/modal.html:6]: You can use the meta author and desc tags here

File: modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            style="width: 716px; height: 429px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline modal/modal.html:92]: Kindly avoid inline styling.

File: slider/slider.css [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+      padding-left: 2rem;
+      
+  
+  }
+  article{
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      /* margin : 2rem 0 8rem 0; */
+  }
```
Comment (bharati-21) [inline slider/slider.css:48]: I see repeated styles here. Saw similar styles in modal.css. Maybe you could combine them?

File: slider/slider.css [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    height: 15px;
+    width: 15px;
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+    transition: background-color 0.6s ease;
+  }
+  .active, .adot:hover {
+    background-color: #717171;
```
Comment (bharati-21) [inline slider/slider.css:199]: You could be consistent and use only variables for colors.

**Review by bharati-21 — APPROVED** [review]
I have dropped a few suggestions. 
I also noticed you used var in your js files. I suggest replacing it with let or const.

Apart from this, looks good to merge. 👍🏽

File: slider/slider.css [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    -webkit-appearance: none;
+    background: transparent;
+    z-index: 99;
+}
+
+.slider input::-webkit-slider-thumb{
+    -webkit-appearance: none;
+    width: 2rem;
+    height: 2rem;
+    background-color: #fff;
```
Comment (yashpatel1797) [inline slider/slider.css:81]: use variable here as well

File: modal/modal.css (hunk 1 above)
Comment (agrawalshanu708) [reply modal/modal.css:76]: thanx bharati, i will sure look into this

File: slider/slider.css [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+    border-radius: 1rem;
+}
+.fwd-icon:hover{
+    background-color: var(--background-color);
+    border-radius: 1rem;
+}
+.dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
```
Comment (yashpatel1797) [inline slider/slider.css:156]: use 'rem' as you have used everywhere.

**Review by yashpatel1797 — APPROVED** [review]
Looks good.

**Pull request merged**